### ISSUE TITLE
Section 3.3.1 is not clear

### ISSUE THREAD

**Issue by thomas-fossati** [ietf-rats-wg/draft-ietf-rats-msg-wrap#104]
Carl, about #78 :

> _'I read the PR but can't say that it helps much. The phrase "If a label changes Verifier state beyond a "hint"" must be evaluated from the point of view of a verifier, right? If different verifiers may elect to use or ignore a "hint", I don't see how this could be any other way. The rest of that sentence establishes that how a verifier acts upon a "hint" determines whether a label is considered to carry semantic content. This impacts the next sentence, which states that "the collection SHOULD be signed by an attestation key" when a label carries semantic content.  Maybe stating "if an attester expects for a label to change Verifier state beyond a "hint"" would salvage the paragraph. Separately, is "label" right here or should it be "item" or "message"? It might also help if "hint" were defined in this context as well as how the "hint" notion interacts with the last point made in the security considerations section.'_

**Comment by nedmsmith** [comment]
"hint" refers to `text`in the `json-collection` and `(int/text)` in the `cbor-collection`. 

We didn't provide CDDL tagging to help with semantics. Possibly we could define them as:
```
json-collection = {
  ? "__cmwc_t": ~uri / oid
  + hint: text => json-CMW / c2j-tunnel
}

cbor-collection = {
  ? "__cmwc_t": ~uri / oid
  + hint: (int / text) => cbor-CMW / j2c-tunnel
}
```

then the prose in §3.3.1 would be more easily connected for the reader.

The prose might be improved by:

A CMW Collection's tree structure is not required to be a spanning tree of the system's composite Attester topology. If the hint includes semantic content for a Verifier (e.g., for better Verifier performance or human comprehension), that is not bound to Evidence contained in the collection, the collection SHOULD be integrity protected, e.g., by including the hint information in a signed Evidence Claim such as EAT [[I-D.ietf-rats-eat](https://ietf-rats-wg.github.io/draft-ietf-rats-msg-wrap/draft-ietf-rats-msg-wrap.html#I-D.ietf-rats-eat)].

**Comment by thomas-fossati** [comment]
> "hint" refers to `text`in the `json-collection` and `(int/text)` in the `cbor-collection`.
> 
> We didn't provide CDDL tagging to help with semantics. Possibly we could define them as:
> 
> ```
> json-collection = {
>   ? "__cmwc_t": ~uri / oid
>   + hint: text => json-CMW / c2j-tunnel
> }
> 
> cbor-collection = {
>   ? "__cmwc_t": ~uri / oid
>   + hint: (int / text) => cbor-CMW / j2c-tunnel
> }
> ```
> 
> then the prose in §3.3.1 would be more easily connected for the reader.

We already use the term "label" for that, we should just use that consistently.

**Comment by nedmsmith** [comment]
> We already have the term "label" to refer to that, we should just use it consistently.

I didn't see `label` in the CDDL. That is the part that is potentially confusing to readers IMO.

Does this CDDL work:
```
json-collection = {
  ? "__cmwc_t": ~uri / oid
  + label: text => json-CMW / c2j-tunnel
}

cbor-collection = {
  ? "__cmwc_t": ~uri / oid
  + label: (int / text) => cbor-CMW / j2c-tunnel
}
```

**Comment by thomas-fossati** [comment]
> > We already have the term "label" to refer to that, we should just use it consistently.
> 
> I didn't see `label` in the CDDL. That is the part that is potentially confusing to readers IMO.
> 
> Does this CDDL work:
> 
> ```
> json-collection = {
>   ? "__cmwc_t": ~uri / oid
>   + label: text => json-CMW / c2j-tunnel
> }
> 
> cbor-collection = {
>   ? "__cmwc_t": ~uri / oid
>   + label: (int / text) => cbor-CMW / j2c-tunnel
> }
> ```

I think you need to do:
```
@@ -1,9 +1,9 @@
 json-collection = {
   ? "__cmwc_t": ~uri / oid
   + &(label: text) => json-CMW / c2j-tunnel
 }

 cbor-collection = {
   ? "__cmwc_t": ~uri / oid
   + &(label: (int / text)) => cbor-CMW / j2c-tunnel
 }
```

**Comment by nedmsmith** [comment]
fixed by PR #106
